release/build/main.go

```diff
@@ -91,13 +92,15 @@ func hashrelaseSubCommands(cfg *config.Config, runner *registry.DockerRunner) []
 			Name:  "publish",
 			Usage: "Publish hashrelease from output/ to hashrelease server",
 			Flags: []cli.Flag{
-				&cli.BoolFlag{Name: skipValidationFlag, Usage: "Skip pre-publish validation", Value: false},
+				&cli.BoolFlag{Name: skipImageScanFlag, Usage: "Skip sending images to image scan service", Value: false},
```

Comment:
I'm not sure this flag name change really makes sense. What I wanted from the `skip-validation` flag was the ability as a developer to skip over the entire validation step, since if I'm testing changes to the release / hashrelease I probably want to skip anything that happens in here (not just image scanning).

Reply:
i'll restore the flag